src/basic/locale-util.c

```diff
@@ -260,7 +260,10 @@ bool locale_is_valid(const char *name) {
         if (!filename_is_valid(name))
                 return false;
 
-        if (!string_is_safe(name))
+        /* Locales look like: ll_CC.ENC@variant, where ll and
+         * CC are alphabetic, ENC is alphanumeric with dashes,
+         * and variant seems to be alphabetic. */
```

Comment:
btw, we do line breaks at 109ch, see .editorconfig

Reply:
gah GNOME has broken me :face_with_spiral_eyes: